wicket-util/src/main/java/org/apache/wicket/util/resource/XSLTResourceStream.java

```diff
@@ -77,7 +74,7 @@ public XSLTResourceStream(final IResourceStream xsltResource, final IResourceStr
 			javax.xml.transform.Result result = new javax.xml.transform.stream.StreamResult(out);
 
 			// create an instance of TransformerFactory
-			javax.xml.transform.TransformerFactory transFact = javax.xml.transform.TransformerFactory.newInstance();
+			javax.xml.transform.TransformerFactory transFact = getTransformerFactory();
```

Comment:
Calling an overwriteable method from the constructor is an anti-pattern - https://stackoverflow.com/questions/3404301/whats-wrong-with-overridable-method-calls-in-constructors

Reply:
Then I would say that the whole processing in the constructor is not a good behavior as the factory needs to be configured in some cases. I am going to change the implementation so that the user can configure it in a different way.